File: Python/7py 1/fibs_fizz_buzz.py
"""
|--------------------------------------------------------------------------
| Fibonacci's FizzBuzz
|--------------------------------------------------------------------------
|
"""

def fibs_fizz_buzz(n):
   x = [1, 1]
   for i in range(2, n): x.append(x[i - 1] + x[i - 2])
   res = ["FizzBuzz" if i % 15 == 0 else "Fizz" if i % 3 == 0 else "Buzz" if i % 5 == 0 else i for i in x]
   return [1] if n == 1 else res


def fibs_fizz_buzz(n):
   r, a, b = [], 0, 1
   for i in range(n):
      r.append("Fizz" * (not b % 3) + "Buzz" * (not b % 5) or b)
      a, b = b, a + b
   return r


def fibs_fizz_buzz(n):
   lst = [1]
   {lst.append(sum(lst[-2:])) for i in range(1,n)}
   return [(i,('Buzz','Fizz')[i%3==0],"FizzBuzz")[(i%3 + i%5==0) + (i%3==0 or i%5==0)] for i in lst]


print(fibs_fizz_buzz(5), [1, 1, 2, "Fizz", "Buzz"])
print(fibs_fizz_buzz(1), [1])

File: PHP/7php 1/fibs_fizz_buzz.php
<?php
/*
|--------------------------------------------------------------------------
| Fibonacci's FizzBuzz
|--------------------------------------------------------------------------
|
*/

function fibs_fizz_buzz($n) {
   $x = [1, 1];
   for ($i = 2; $i < $n; $i++) $x[] = $x[$i - 1] + $x[$i - 2];
   $res = array_map(function($i) { return $i % 15 == 0 ? "FizzBuzz" : ($i % 3 == 0 ? "Fizz" : ($i % 5 == 0 ? "Buzz" : $i)); }, $x);
   return $n == 1 ? [1] : $res;
}


function fibs_fizz_buzz($n) {
   $fib = [1, 1];
   $fizz = [1, 1];
   while (count($fib) < $n) {
      $new = $fib[count($fib) - 2] + $fib[count($fib) - 1];
      $fib[] = $new;
      $fizz[] = ($new % 3 == 0) ? (($new % 15 === 0) ? "FizzBuzz": "Fizz") : (($new % 5 === 0) ? "Buzz" : $new);
   }
   return array_slice($fizz, 0, $n);
}


function fibs_fizz_buzz($n) {
   $fibs = [0, 1];
   $result = [1];
   for($i=2;$i<=$n;$i++){
      $fibs[$i] = $fibs[$i-2] + $fibs[$i-1];
      if($fibs[$i]%3==0 && $fibs[$i]%5==0) $result[]='FizzBuzz';
      elseif($fibs[$i]%3==0) $result[]='Fizz';
      elseif($fibs[$i]%5==0) $result[]='Buzz';
      else $result[]=$fibs[$i];
   }
   return $result;
}


print_r(fibs_fizz_buzz(5));
print_r(fibs_fizz_buzz(1));